bitnami/thanos/values-production.yaml

```diff
@@ -792,6 +796,10 @@ bucketweb:
     ## It is also possible to create and manage the certificates outside of this helm chart
     ## Please see README.md for more information
     ##
+    ## When specifying cert-manager.io/cluster-issuer: nameOfClusterIssuer annotation, enable tls for ingress
```

Comment:
I would separate this from the previous section. Please add a space before, and the same in the rest of the changes like this one